server/app.js

```diff
@@ -113,8 +114,9 @@ if (process.env.NODE_ENV !== 'production') {
 		cache,
 		apiKey
 	];
-
+	app.post('/syndication/contracts/:contract_id/resolve', [...middleware, getContractByIdFromParam], require('./controllers/resolve'));
```

Comment:
I don't think you'd necessarily need to use the array spread here. I think you could add the getContractByIdFromParam as the 3rd param but 🤷‍♂️ don't care really

```js
app.post(
	'/syndication/contracts/:contract_id/resolve',
	middleware,
	getContractByIdFromParam,
	require('./controllers/resolve')
);
```

Reply:
will ditch the spread

Reply:
it is so ditched